index.html

```diff
@@ -1,143 +1,82 @@
 <!DOCTYPE html>
-<html lang="">
-  <head>
-    <link href="https://fonts.bunny.net/css2?family=Fira+Sans&display=swap" rel="stylesheet">
```

Comment:
You might want to keep this line to ensure the font loads correctly if it isn't installed on the user's system (depending on which font you want the website to use)

Reply:
Switched to Fira Sans